deploy-service/common/src/main/java/com/pinterest/deployservice/handler/PingHandler.java

```diff
@@ -144,6 +126,27 @@ void updateHosts(PingRequestBean pingRequest) throws Exception {
         }
     }
 
+    void updateHostTag(PingRequestBean pingRequestBean) throws Exception {
+        String hostId = pingRequestBean.getHostId();
+        String hostName = pingRequestBean.getHostName();
+        List<Map<String, String>> tags = pingRequestBean.getTags();
+        if(tags != null) {
```

Comment:
maybe better to use (by adding a space):
if (tags != null) {

I also notice similar issues in this PR. I understand that there is no style check in the pom. I'm sorry that it just that my coding habit  prompts me to feel uncomfortable about this :-)